.pre-commit-config.yaml

```diff
@@ -33,7 +33,7 @@ repos:
     - id: pyupgrade
       args: ["--py37-plus"]
 -   repo: https://github.com/nbQA-dev/nbQA
-    rev: 0.5.7
+    rev: v0.5.8-1-g1defb695ced783d6f67273a21bdd1951ea3f7e90
```

Comment:
I'm going to hold off on merging this until we understand the switch to using commit hashes in the `rev`.